demo/Demo/Textfields.elm

```diff
@@ -224,6 +225,32 @@ textfields model =
         ]
        """
     )
+  , ( "Expandable textfield"
+    , Textfield.render Mdl [11] model.mdl
+        [ Textfield.label "Expandable"
+        , Textfield.floatingLabel
+        -- NOTE: The id's must match
```

Comment:
This is the best I can think of: 
- if user sets inner and expandable and they differ, log an error message on the console, use `inner`.
- if they set just one, use that one. 

?